src/Orleans.Core/Runtime/OutsideRuntimeClient.cs

```diff
@@ -252,8 +234,6 @@ await this.gatewayListProvider.InitializeGatewayListProvider()
                 ct).Ignore();
             grainInterfaceMap = await transport.GetTypeCodeMap(this.InternalGrainFactory);
             
-            await ClientStatistics.Start(statisticsProviderManager, transport, clientId)
```

Comment:
please point me to the logic which handles `ClientStatistics.Start` now. `ClientStatisticsManager.Start` init statistic publishers and also start the statistic runtime, which keep a list of internal metrics and also statistic tool classes to calculate the metrics and report them periodically.

Reply:
I likely broke this.  Will look.